src/ImageSharp/Common/Helpers/Guard.cs

```diff
@@ -14,7 +14,7 @@ namespace ImageSharp
     /// Provides methods to protect against invalid parameters.
     /// </summary>
     [DebuggerStepThrough]
-    internal static class Guard
+    public static class Guard
```

Comment:
yeah my bad, that shouldn't be public, I had an attempt to make it compile that needed it, must have forgot to change it back.